### PR TITLE
WAVE2 1048 Grad details supervisor view

### PR DESCRIPTION
**Pull request by robbdavis**
WAVE2 1048 Grad details supervisor view fe integration

### CODE REVIEW

**Comment by dpfitzsi** [comment]
Robb can you resolve the conflicts now that the fgsmerge branch has been moved to develop?

**Comment by dpfitzsi** [comment]
Robb, in this branch can you rebuild the CSS files (maps, scss, etc.) for the ucgy_fgs module. I wouldn't be surprised it they're not perfect.

File: sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_supervisor/ucgy_gbl_supervisor.features.field_instance.inc [hunk 1]
```diff
@@ -325,8 +325,8 @@ function ucgy_gbl_supervisor_field_default_field_instances() {
     'bundle' => 'gbl_supervisor',
     'default_value' => NULL,
     'deleted' => 0,
-    'description' => 'Abbreviations (letters) of your degree credentials to appear beside your name.<br>
-For example: PhD, MSc, MA, etc...',
+    'description' => 'Abbreviations (letters) of your degree credentials to appear beside your name.<br>
+For example: MBA, Phd, MSc, etc...',
```
Comment (mhughes2012) [inline sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_supervisor/ucgy_gbl_supervisor.features.field_instance.inc]: MBA was removed from this list in jira [wave2-1017]

File: sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_supervisor/ucgy_gbl_supervisor.features.field_instance.inc [hunk 2]
```diff
@@ -927,8 +927,8 @@ For example: PhD, MSc, MA, etc...',
   // Translatables
   // Included for use with string extractors like potx.
   t('<strong>9 Maximum</strong>');
-  t('Abbreviations (letters) of your degree credentials to appear beside your name.<br>
-For example: PhD, MSc, MA, etc...');
+  t('Abbreviations (letters) of your degree credentials to appear beside your name.<br>
+For example: MBA, Phd, MSc, etc...');
```
Comment (mhughes2012) [inline sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_supervisor/ucgy_gbl_supervisor.features.field_instance.inc]: same here.

File: sites/all/modules/ucalgary/ucgy_fgs/js/fgs_filters.js [hunk 3]
```diff
@@ -0,0 +1,275 @@
...
+
+      // $(".program .grad-program-tile").on("click", function(event){
+      //     // Only if NOT clicking on actual link - this prevents ctrl/command + click
+      //     // shortcut from spawning new window and also loading link in existing page
+      //     // Also need to check if click action is for the comparison star or not...
+      //     if ( !$( event.target ).is( "a" ) && !$( event.target ).is( "button" ) ) {
+      //         location.href = $("p.degree > a", $(this)).attr('href');
+      //     }
+      // });
+
```
Comment (robbdavis) [inline sites/all/modules/ucalgary/ucgy_fgs/js/fgs_filters.js]: Same as above. If pulling out this code is required (looks like it should be pulled out) that should be documented elsewhere.

File: sites/all/modules/ucalgary/ucgy_fgs/js/fgs_filters.js [hunk 4]
```diff
@@ -0,0 +1,275 @@
...
+        // $(".related-programs-list").each(function(){
+        //   if ($(".programs > .program", $(this)).not('.minimized').size() == 0) {
+        //     $(".programs", $(this)).addClass("empty");
+        //   }
+        //   else {
+        //     $(".programs", $(this)).removeClass("empty");
+        //   }
+        // });
+      }
+
```
Comment (robbdavis) [inline sites/all/modules/ucalgary/ucgy_fgs/js/fgs_filters.js]: Hi Mike. I never touched this file as part of this story. It may appear changed because of a merge. You should only check the commits by me that are not merges. Changing this here is out of scope.

File: sites/all/modules/ucalgary/ucgy_fgs/templates/node--fgs-graduate-degree.tpl.php [hunk 5]
```diff
@@ -9,408 +9,602 @@
...
+                        <h1><?php echo $grad_program; ?></h1>
+                        <h4><?php echo $grad_degree . ' ('. $grad_abbr .')'; ?></h4>
+                        <h4><?php echo render($content['field_fgs_grad_degree_faculty']);?></h4>
+                        <p class="compare">
+                            <button class="btn btn-rounded compare-fgs-program" type="button" data-id="<?php print $node->nid; ?>">
+                                <span class="sr-only">Toggle Compare</span>
+                                <span class="glyphicon glyphicon-star-empty"></span>
+                            </button>
+                            Add program to comparison list
+                        </p>
```
Comment (robbdavis) [inline sites/all/modules/ucalgary/ucgy_fgs/templates/node--fgs-graduate-degree.tpl.php]: Discussed this with Dan previously. Translation was agreed to not be a requirement of this project early on.Generally, we should try to use the t function when it is easy to do so. (might as well right? just in case it is one day required)

But in a large template like this, it is not required and not efficient to call the t function all over the place.

File: sites/all/modules/ucalgary/ucgy_fgs/templates/node--fgs-graduate-degree.tpl.php [hunk 6]
```diff
@@ -9,408 +9,602 @@
...
+    <div class="fieldable-panels-pane container-fluid brick fgs-program-detail-page contextual-links-region outline-text core-courses-list red-lines">
+        <div class="row">
+            <div class="container">
+                <?php
+                //see if author set any value for core course
+                if ($content['field_fgs_core_courses']['#items'][0] != '') {
+                    ?>
+                    <div class="row header">
+                        <div class="col-sm-12">
+                            <div class="icon"><img class="img-responsive" src="http://placehold.it/56x56"></div>
```
Comment (robbdavis) [inline sites/all/modules/ucalgary/ucgy_fgs/templates/node--fgs-graduate-degree.tpl.php]: This has be resolved in the big 910-914-974etc.. branch.

File: sites/all/modules/ucalgary/ucgy_fgs/templates/node--fgs-graduate-degree.tpl.php [hunk 7]
```diff
@@ -9,408 +9,602 @@
...
+?>
+
+<div class="fieldable-panels-pane container-fluid brick fgs-program-detail-page contextual-links-region outline-text admission-requirements-list ruby-lines">
+    <div class="row">
+        <div class="container">
+            <div class="row header">
+                <div class="col-sm-12">
+                    <?php // <div class="icon"><span class="glyphicon glyphicon-ok-sign" aria-hidden="true"></span></div> ?>
+                    <div class="icon"><img class="img-responsive" src="http://placehold.it/56x56"></div>
+                    <div class="text">
```
Comment (robbdavis) [inline sites/all/modules/ucalgary/ucgy_fgs/templates/node--fgs-graduate-degree.tpl.php]: as above

File: sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_supervisor/ucgy_gbl_supervisor.features.field_instance.inc (hunk 1 above)
Comment (robbdavis) [inline sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_supervisor/ucgy_gbl_supervisor.features.field_instance.inc]: I think this is being resolved in story 1064

File: sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_supervisor/ucgy_gbl_supervisor.features.field_instance.inc (hunk 2 above)
Comment (robbdavis) [inline sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_supervisor/ucgy_gbl_supervisor.features.field_instance.inc]: as above
